src/Kobo/PowerOff.cpp

```diff
@@ -84,9 +85,18 @@ DrawBanner(Canvas &canvas, PixelRect &rc)
   canvas.DrawText({x, rc.top + int(banner_height - normal_font.GetHeight()) / 2},
                   website);
 
-  const TCHAR *const comment = _T("powered off");
-  canvas.DrawText({rc.right - (int)canvas.CalcTextWidth(comment) - padding, rc.top + padding},
-                  comment);
+  TCHAR battery[4];
+  if (File::ReadString(Path("/sys/class/power_supply/mc13892_bat/capacity"), battery, sizeof(battery))){
```

Comment:
This duplicates code from `Hardware/Battery.cpp` - if you need this, better use that library; the library is hard to use, because it writes to global variables, which is bad, but you can move the Kobo specific part to a separate library e.g. `Hardware/KoboBattery.cpp` in a separate commit, and amend this PR's existing commit to use that library.

Reply:
I'm afraid this is way over my skills :-(

Reply:
OK I'll do that.